db/migrate/20220721003940_create_user_groups.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUserGroups < ActiveRecord::Migration[6.0]
+  def change
+    create_table :user_groups do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :group, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    #add_index :user_groups, [ :user_id, :group_id ], unique: true
```

Comment:
Awesome! I think we can uncomment this like, makes total sense to me!

```suggestion
    add_index :user_groups, [ :user_id, :group_id ], unique: true
```